spec/request/mutations/edit_user_spec.rb

```diff
@@ -39,4 +39,27 @@ def query(id)
     json = JSON.parse(response.body)
     expect(json['data']['editUser']['errors']).to include("Couldn't find User with 'id'=0")
   end
+
+  xit 'returns error if user cannot be updated' do
```

Comment:
if we are not using this test, should we remove it?

Reply:
I guess i could have deleted it